package.json

```diff
@@ -41,6 +42,7 @@
     "react-device-detect": "^1.17.0",
     "react-dom": "^17.0.2",
     "react-intl": "^5.17.6",
+    "react-popper": "^2.2.5",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
     "react-tippy": "^1.4.0",
```

Comment:
since we are replacing `react-tippy` we can remove this dependency

Reply:
yeah was thinking about it, done!